### PR TITLE
Azure Node Commerce Service

### PR DESCRIPTION
**Pull request by jawache**

### CODE REVIEW

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/commerce.md](https://github.com/jawache/azure-docs-sdk-node/blob/patch-19/docs-ref-services/commerce.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C3%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201708031848141895-190%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/commerce.md](https://github.com/jawache/azure-docs-sdk-node/blob/patch-19/docs-ref-services/commerce.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C3%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201708031848560709-190%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/commerce.md](https://github.com/jawache/azure-docs-sdk-node/blob/patch-19/docs-ref-services/commerce.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C3%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201708031857418458-190%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/commerce.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Azure Commerce modules for Node.js
-description: Reference for Azure Commerce modules for Node.js
+title: Azure Commerce Modules for Node.js
```
Comment (johnpapa) [inline docs-ref-services/commerce.md]: @burkeholland  says lowercase all Ms now. sorry.

**Comment by johnpapa** [comment]
#sign-off 

once the M is fixed

File: docs-ref-services/commerce.md (hunk 1 above)
Comment (johnpapa) [inline docs-ref-services/commerce.md]: @jawache ^^

**Comment by jawache** [comment]
M is fixed!

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/commerce.md](https://github.com/jawache/azure-docs-sdk-node/blob/patch-19/docs-ref-services/commerce.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C4%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201708041511449009-190%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by johnpapa** [comment]
#sign-off

**Comment by jawache** [comment]
Actually #hold-off on this one, I've got a question about billing vs commerce!

**Comment by jawache** [comment]
John Lian <jlian@microsoft.com> has confirmed that this overview is for the commerce API and not for the billing API. Changed the wording to match.

 #sign-off again.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/commerce.md](https://github.com/jawache/azure-docs-sdk-node/blob/patch-19/docs-ref-services/commerce.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C8%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201708081519075979-190%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**